VAGRANT.md

```diff
@@ -44,3 +47,7 @@ rails server --binding=0.0.0.0
 You edit the code on your computer using the code editor you are used to using, then through shared folders the code is updated on the VM instantly.
 
 You should run the tests before submitting any patch or Pull Request back to the original repository. Please see [CONTRIBUTING.md](CONTRIBUTING.md) for more information.
+
+## Troubleshooting
+
+If you have issues installing ssh tunnel to the environment, try adding `:nfs_udp => false` to `override.vm.synced_folder` options in the `Vagrantfile`.
```

Comment:
It uses TCP by default, but for the libvirt provider it may fail with some configurations. I did not manage to google the fix for Fedora Linux, but found several solutions that suggested using the UDP transport.

I didn't do this for virtualbox, since it worked with the default settings and TCP transport.